twitter/search.go

```diff
@@ -49,6 +49,7 @@ type SearchTweetParams struct {
 	MaxID           int64  `url:"max_id,omitempty"`
 	Until           string `url:"until,omitempty"`
 	IncludeEntities *bool  `url:"include_entities,omitempty"`
+	TweetMode       string `url:"tweet_mode,omitempty"`
```

Comment:
I'm not seeing it mentioned at https://dev.twitter.com/rest/reference/get/search/tweets, am I looking in the wrong place?